jaeger-core/src/test/java/io/jaegertracing/internal/JaegerSubclassTest.java

```diff
@@ -0,0 +1,179 @@
+package io.jaegertracing.internal;
+
+import io.jaegertracing.internal.clock.Clock;
+import io.jaegertracing.internal.metrics.Metrics;
+import io.jaegertracing.spi.BaggageRestrictionManager;
+import io.jaegertracing.spi.Reporter;
+import io.jaegertracing.spi.Sampler;
+import io.opentracing.Scope;
+import io.opentracing.ScopeManager;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class JaegerSubclassTest {
+  private static class Configuration extends io.jaegertracing.Configuration {
```

Comment:
`s/Configuration/CustomConfiguration/` (or anything else that won't class with i.j.Configuration)

Reply:
Right good idea.